frontend/src/renderer/components/Process/Dropbox.svelte

```diff
@@ -9,27 +9,30 @@
     export let items = [];
     export let type;
     export let singleItem = true;
-    export let showError = false;
+    export let minimum = true;
 
     function handleSort(e) {
         items = e.detail.items;
     }
-
 </script>
-<main>
-    <div class="sectionStyle {showError && singleItem && isEmpty(items) ? 'errorStyle' : ''}">
-        <section use:dndzone={{items, flipDurationMs, type, dropFromOthersDisabled: singleItem && items.length > 0}}
-            on:consider={handleSort}
-            on:finalize={handleSort}
-        >
-            {#each items as item (item?.id)}
-                <card animate:flip={{ duration: flipDurationMs }} class="{singleItem ? 'verified' : ''}">
+
+<div class="sectionStyle {singleItem && isEmpty(items) ? 'errorStyle' : ''}">
+    <section use:dndzone={{items, flipDurationMs, type, dropFromOthersDisabled: singleItem && items.length > 0}}
+        on:consider={handleSort}
+        on:finalize={handleSort}
+    >
+        {#each items as item (item?.id)}
+            <card animate:flip={{ duration: flipDurationMs }} class="{singleItem ? 'verified' : ''}">
+                {#if minimum}
```

Comment:
what exactly is this 'minimum' flag for?

Reply:
The minimum flag is the difference in the way the file is displayed in the uploader section vs file roles.  It allows us to have the little image icon along with the trash icon to remove an individual image.